src/builtins/copy-param/index.ts

```diff
@@ -0,0 +1,78 @@
+import {z} from 'zod';
+import {ERRORS} from '@grnsft/if-core/utils';
+import {ExecutePlugin, PluginParams} from '@grnsft/if-core/types';
+
+import {validate} from '../../util/validations';
+
+import {STRINGS} from '../../config';
+
+const {MISSING_GLOBAL_CONFIG, MISSING_INPUT_DATA} = STRINGS;
+const {GlobalConfigError, MissingInputDataError} = ERRORS;
+// global-config:
```

Comment:
if we want to keep this comment here, it can be done like this

/**
 * <your description here>
 */
 export const Copy ...

Reply:
```suggestion
```